testdata/p4_16_psa_errors_outputs/psa-example-dpdk-directmeter-err.p4-error

```diff
@@ -1,3 +1,3 @@
-psa.p4(638): [--Werror=unexpected] error: execute method of meter0 extern can only be invoked from within action of ownertable
+psa.p4(657): [--Werror=unexpected] error: execute method of meter0 extern can only be invoked from within action of ownertable
```

Comment:
The method name should have been dpdk_execute

Reply:
Can we get rid of hardcode method names from error messages?